pkg/provider/azure_storageaccount.go

```diff
@@ -277,7 +277,7 @@ func (az *Cloud) EnsureStorageAccount(accountOptions *AccountOptions, genAccount
 }
 
 func (az *Cloud) createPrivateEndpoint(ctx context.Context, accountName string, accountID *string, privateEndpointName string) error {
-	klog.V(5).Infof("Creating private endpoint(%s) for account (%s)", privateEndpointName, accountName)
+	klog.V(2).Infof("Creating private endpoint(%s) for account (%s)", privateEndpointName, accountName)
```

Comment:
this is actually decreasing the log level

Reply:
make it print by default in csi driver